repositories/like_repository.py

```
@@ -0,0 +1,63 @@
+        if not os.path.exists(self.file_path):
+            self._save_data([])
+
+    def _load_data(self) -> List[dict]:
+        """JSON 파일에서 좋아요 목록 로드"""
+        try:
+            with open(self.file_path, "r", encoding="utf-8") as f:
+                return json.load(f)
+        except (FileNotFoundError, json.JSONDecodeError):
+            return []
+
+    def _save_data(self, data: List[dict]):
```

Ruff (0.14.13)

Missing return type annotation for private function `_save_data`

Add return type annotation: `None`

(ANN202)